### DISCUSSION TITLE
Camera hardware confusion

### DISCUSSION THREAD

**Discussion by labaxisAI2** [AxisCommunications/acap-computer-vision-sdk-examples#185] [General] [answered]
I am curious about my Q1615-LE Mk3 box camera. 
The product should have a ARTPEC chip with a DLPU, this is similar to my Q3628-VE Dome Camera (perhaps a different ARTPEC version)
[https://www.axis.com/products/axis-q1615-le-mk-iii#technical-specifications](url)

I was able to run the pose-estimator with flask on both cameras, however I noticed that my _box_ camera received an error:
`The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/arm/v7) and no specific platform was requested`
When I change my scripts to the `ARCH=armv7hf`, and `env.armv7hf.tpu` config, the _box_ camera runs the pose-estimator well.  I thought this was odd since the chip and unit was artpec and DLPU.

My main issue is now I am receiving a fatal error attempting to run my Custom .tflite model (yolov5)  `Could not load model: Model is not compiled for EdgeTPU`.  I was able to adapt the pose estimator setup/scripts to run a yolov5 model almost flawlessly on the _dome_ camera.

Have I overlooked something while setting up my camera or environment? 

**Edit:** The box camera with `env.armv7hf.cpu` config "works", however inference speed is very slow at 1600ms. So does not take advantage of DLPU and artpec chip

**Comment by Corallo** [comment, marked as answer]
Hi @labaxisAI2 

You can find more information about the products [here](https://www.axis.com/en-us/support/tools/product-selector)
The difference is that Q1615-LE Mk3 is an Artpec-7, while Q3628-VE is an Artpec-8.
Those two families have both different architecture and DLPU capability.
I see you already solved the architecture problem by rebuilding for armv7hf.

Regarding the model compatibility, you can see [here](https://axiscommunications.github.io/acap-documentation/docs/computer-vision-on-device/axis-deep-learning-procesing-unit.html) and [here](https://axiscommunications.github.io/acap-documentation/docs/computer-vision-on-device/dlpu-model-conversion.html) for more details on those DLPUs.
Yolov5 has an [export flag](https://github.com/ultralytics/yolov5/blob/master/export.py#L16) to obtain a edgeTPU model (compatible with Artpec-7) but expect worse performance compared to Artpec-8.

**Reply by labaxisAI2** [reply]
Thank you for the information, I was using [this](https://www.axis.com/products/axis-q1615-le-mk-iii) as reference and that page was not as clear as the link you provided.
